viewpump/src/main/res/values/ids.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <item name="viewpump_tag_id" type="id"/>
+</resources>
```

Comment:
np at all, will add. we can get a basic checkstyle added in a future PR and run it on Travis